### DISCUSSION TITLE
YOLO Badge appears to be given for the wrong reason

### DISCUSSION THREAD

**Discussion by cajubelt** [community/community#41662] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For me it says I merged a PR without a review, but when you look at the PR, it has an approval before I merged it.

https://github.com/cajubelt?achievement=yolo&tab=achievements

**Comment by KDA809** [comment]
The only requirement to earn badge is to Merge without reviewing so you fulfilled requirement :)

**Comment by rameshkumarkhatri** [comment]
I saw a profile that he merged the pr after two approval and still got the badge. On badge it referred to PR, I open the PR and it has two approvals.

**Comment by DGuhr** [comment]
my 2c: I find the badge a bit misleading in it's message, too, because it does not take into account other programming models like Trunk based development, pairing and ensemble programming. 

It might be me, but it feels somewhat rude to say I am a YOLO person when I do pair- and/or mob programming, and as such continuous code review, where the whole "async pull request approval after async code review" thing that I find very inefficient is just put as a given of how one "should" do it, just because it's the standard on this platform (for better or worse).  I got this badge now that sets it into the context of someone who is kinda irresponsible. Dislike.

**Reply by jamesreprise** [reply]
> [...] I got this badge now that sets it into the context of someone who is kinda irresponsible. Dislike.

You can hide badges by clicking on them then toggling their visibility by clicking the eye icon on the top left of the popup.

**Reply by DGuhr** [reply]
good to know as a workaround, thx :)

**Comment by RONIE70** [comment]
Cree una nueva solicitud de extracción en su repositorio y ábrala
En la esquina superior derecha de la sección Revisores, solicite a alguien que revise su solicitud de extracción
Combinar la solicitud de extracción sin finalizar la revisión.
Puede consultar mi solicitud de extracción en [este repositorio](https://github.com/ratul-hsan/Current-Activity/pull/10) que me dio la insignia de logros de YOLO.

**Comment by QuinSDev** [comment]
> ### Seleccionar área temática
> Comentarios sobre el producto
> 
> ### Cuerpo
> Para mí, dice que fusioné un PR sin una revisión, pero cuando miras el PR, tiene una aprobación antes de fusionarlo.
> 
> [https://github.com/cajubelt?achievement=yolo&tab=logros](https://github.com/cajubelt?achievement=yolo&tab=achievements)

Es verdad!!
